src/components/tabs/components/TabItem.vue

```diff
@@ -88,7 +96,56 @@ export default {
             border-color: $vs-color-interaction-cta-disabled;
         }
 
-        &:focus-visible{
+        &:focus-visible {
+            box-shadow: $vs-box-shadow-focus;
+            outline: none;
+        }
+    }
+
+    .vs-tab-link--bsh {
+        flex-grow: 1;
+        background-color: $vs-color-interaction-cta-subtle-hover;
+        color: $vs-color-text-primary;
+        border-width: 1px;
+        border-color: $vs-color-border-primary;
+        border-radius: 5px 5px 0 0;
```

Comment:
These should probably use tokens tokens (border_radius_lg should match up to 5, or close to it, along with spacer-0)